phpcs.xml.dist

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress Theme Coding Standards">
+	<!-- See https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml -->
+	<!-- See https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/develop/WordPress-Core/ruleset.xml -->
+
+	<!-- Set a description for this ruleset. -->
+	<description>A custom set of code standard rules to check for WordPress themes.</description>
+
+	<!-- Include the WordPress ruleset, with space for exclusions if necessary. -->
+	<rule ref="WordPress-Core">
+		<exclude name="Generic.WhiteSpace.ScopeIndent.Incorrect" />
+		<exclude name="Generic.WhiteSpace.ScopeIndent.IncorrectExact" />
+
+		<exclude name="PEAR.Functions.FunctionCallSignature.Indent" />
+
+		<exclude name="Squiz.Commenting.FileComment.SpacingAfterComment" />
+		<exclude name="Squiz.Commenting.FunctionComment.MissingParamTag" />
+		<exclude name="Squiz.Commenting.InlineComment.InvalidEndChar" />
+		<exclude name="Squiz.Commenting.InlineComment.NotCapital" />
+	</rule>
+	<rule ref="WordPress-Docs">
+
+	</rule>
+
+	<rule ref="Squiz.Commenting.FunctionComment.ScalarTypeHintMissing">
+		<severity>0</severity>
+	</rule>
+</ruleset>
```

Comment:
Do we want to use core ruleset because it's a bundled theme? Since it is a theme, it would be good if we could use [WPThemeReview](https://github.com/WPTRT/WPThemeReview/) ruleset (plus any needed modifications if needed). We just need to release the theme review ruleset (0.1.0 should be released soon)

Reply:
Good point, yes, it'd be great to be able to use WPThemeReview's ruleset 🎉

FYI: The WPThemeReview ruleset also requires WPCS `1.0.0` 
https://github.com/WPTRT/WPThemeReview/#requirements

Reply:
@ntwb be my guest (or my host) to amend this PR as you feel best.